aws_sqs_entity.api.php

```diff
@@ -13,18 +13,26 @@
 /**
  * Allows modules to respond after an item is sent to the AWS Queue.
  *
- * @param string $type
- *   The Entity type.
- * @param object $entity
- *   The Entity object.
- * @param string $op
- *   The Entity CRUD operation. Can be one of:
- *   - insert
- *   - update
- *   - delete
+ * @param array $item_info
+ *   An array of contextual data with the following keys:
+ *     'type' => Entity type
+ *     'entity' => Entity object
+ *     'op' => Operation: 'insert', 'update', or 'delete'
+ *     'result' => Boolean representing success or failure
+ *     'data' => The message data
+ *     'message' => [
+ *       'attributes' => SQS Message attributes
+ *       'id' => SQS Message ID
+ *     ],
+ *     'error' => [ [ONLY PRESENT IN ERROR CASES]
+ *       'response_message' => Response message
+ *       'response_code' => Response code
+ *       'response_raw' => Raw response data
+ *     ],
  */
-function hook_aws_sqs_entity_send_item($type, $entity, $op) {
+function hook_aws_sqs_entity_send_item($item_info) {
```

Comment:
Nice 👍  (see note below at invocation). PS I'm not concerned about a breaking change here, since we're still pre-release in GitHub (and not the only project using this hasn't used the notification hook yet).